samples/matter/common/src/bridge/ble_connectivity_manager.cpp

```diff
@@ -142,6 +176,98 @@ void BLEConnectivityManager::DisconnectionHandler(bt_conn *conn, uint8_t reason)
 	}
 }
 
+#if defined(CONFIG_BT_SMP)
+void BLEConnectivityManager::SecurityChangedHandler(struct bt_conn *conn, bt_security_t level, enum bt_security_err err)
+{
+	if (err) {
+		LOG_ERR("Security failed: level %d err %d", level, err);
+	} else {
+		LOG_INF("Security changed: level %d", level);
+	}
+}
+
+void BLEConnectivityManager::AuthenticationCancel(struct bt_conn *conn)
+{
+	char addr[BT_ADDR_LE_STR_LEN];
+
+	bt_addr_le_to_str(bt_conn_get_dst(conn), addr, sizeof(addr));
+
+	LOG_ERR("Pairing cancelled: %s\n", addr);
+
+	bt_addr_le_t a = *bt_conn_get_dst(conn);
```

Comment:
Can we select more descriptive variable name? Or just do:
`Instance().RemoveBLEProvider(*bt_conn_get_dst(conn));`